block_visualizer_plugin/src/template/block_visualizer_template.jinja2

```diff
@@ -437,6 +435,8 @@
             d.fy = null;
 	    updateBirdView();
         }
+        birdView.on("click", function (event) {
+});
```

Comment:
This seems to be an empty function :thinking:

Reply:
It allows bird view to be sensitive on click, but it doesn't specifically do until I click on nodes whose method is separated. If I don't have it, the view, won't recognize the click